Earthfile

```diff
@@ -341,10 +341,10 @@ earthly:
     ARG GOCACHE=/go-cache
     RUN mkdir -p build
     RUN printf "$BUILD_TAGS" > ./build/tags && echo "$(cat ./build/tags)"
-    RUN printf '-X main.DefaultBuildkitdImage='"$DEFAULT_BUILDKITD_IMAGE" > ./build/ldflags && \
-        printf ' -X main.Version='"$VERSION" >> ./build/ldflags && \
-        printf ' -X main.GitSha='"$EARTHLY_GIT_HASH" >> ./build/ldflags && \
-        printf ' -X main.DefaultInstallationName='"$DEFAULT_INSTALLATION_NAME" >> ./build/ldflags && \
+    RUN printf '-X github.com/earthly/earthly/cmd/earthly/flag.DefaultBuildkitdImage='"$DEFAULT_BUILDKITD_IMAGE" > ./build/ldflags && \
```

Comment:
Can we avoid changing this?

They need to be kept in sync with https://github.com/Homebrew/homebrew-core/blob/master/Formula/earthly.rb#L25-L29 and it's easier if we don't have to submit a PR to homebrew.

as well as in our own homebrew tap, which is linked to below under the `# Important! ...` comment.